packages/now-build-utils/src/detect-routes.ts

```diff
@@ -265,11 +266,27 @@ async function detectApiRoutes(
   }
 
   // 404 Route to disable directory listing
-  if (defaultRoutes.length) {
-    defaultRoutes.push({
-      status: 404,
-      src: '/api(\\/.*)?$',
-    });
+  if (defaultRoutes.length > 0) {
+    if (featHandleMiss) {
+      defaultRoutes = [
+        { handle: 'miss' },
+        {
+          src: '/api/(.+)\\.\\w+',
+          dest: '/api/$1',
+          check: true,
+        },
+        {
+          status: 404,
+          src: '/api(\\/.*)?$',
```

Comment:
We don't need to escape the `/` here or below

Reply:
Thanks, I removed the superfluous escape sequence